cdc/sink/dmlsink/mq/manager/kafka_manager.go

```diff
@@ -102,20 +102,10 @@ func (m *kafkaTopicManager) backgroundRefreshMeta(ctx context.Context) {
 			)
 			return
 		case <-m.metaRefreshTicker.C:
-			topicMetaList, err := m.getMetadataOfTopics(ctx)
-			// We ignore the error here, because the error may be caused by the
-			// network problem, and we can try to get the metadata next time.
-			if err != nil {
-				log.Warn("Get metadata of topics failed",
-					zap.String("namespace", m.changefeedID.Namespace),
-					zap.String("changefeed", m.changefeedID.ID),
-					zap.Error(err))
+			topicPartitionNums, _ := m.fetchAllTopicsNumPartitions(ctx)
```

Comment:
Why don't we modify this method to not return an error, and add a comment explaining why the error can be safely ignored in the method?